projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/payment-methods.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Payment Methods Page accessibility', { testIsolation: false }, () => {
+  before(() => {
+    cy.a11yContinuumSetup();
+    cy.login('test-user-with-orders@sap.cx.com', 'pw4all');
```

Comment:
We should import the credentials from local sample data files. There should be examples of this in other tests.